src/System.Private.CoreLib/shared/System/Double.cs

```diff
@@ -217,47 +246,47 @@ public unsafe override int GetHashCode()
         public override String ToString()
         {
             Contract.Ensures(Contract.Result<String>() != null);
-            return FormatProvider.FormatDouble(m_value, null, null);
+            return Number.FormatDouble(m_value, null, NumberFormatInfo.CurrentInfo);
```

Comment:
I've added a commit that moves `System.Globalization.FormatProvider.Number` to just be `System.Number`.

Will revert and use the `#if` defs if it causes any significant problems in the PR jobs.